### PR TITLE
Place hash

### LINKED ISSUE #426: Twitter geodata bounding box makes places viewer show weird results
For geotagged tweets with bounding box, we should probably find the center of the bounding box.  right now we just grab the first box corner and use that, putting some location markers in the middle of the ocean.

### PR DESCRIPTION
**Pull request by amcjen**
- Fix ID handling for places collection.
- Normalize all lat/lng values to a fixed 5 decimal places
- Fix #426: bounding box weirdness for Twitter

### CODE REVIEW

File: Collections/Places/dataStore.js [hunk 1]
```diff
@@ -67,19 +66,19 @@ function processTwitter(svcId, type, data, cb) {
...
-    
+
     var ll = firstLL(data.geo);
     if (!ll) {
-        ll = firstLL(data.place, true);
-    }
-    if (!ll) {
         ll = firstLL(data.coordinates, true);
     }
+    if (!ll) { // bounding box.  Compute the center
```
Comment (ctide) [inline Collections/Places/dataStore.js]: can't you just do:

var ll = firstLL(data.geo) || firstLL(data.coordinates, true) || computedLL(data.place.bounding_box.coordinates[0]); ?

File: Collections/Places/dataStore.js [hunk 2]
```diff
@@ -67,19 +66,19 @@ function processTwitter(svcId, type, data, cb) {
...
+        ll = computedLL(data.place.bounding_box.coordinates[0]);
+    }
     if (!ll) {
         // quietly return, as lots of tweets aren't geotagged, so let's just bail
         cb();
```
Comment (ctide) [inline Collections/Places/dataStore.js]: This should be:

return cb();

Right?

File: Collections/Places/dataStore.js (hunk 1 above)
Comment (amcjen) [inline Collections/Places/dataStore.js]: yeeees, likely could!  Will fix this up.

File: Collections/Places/dataStore.js (hunk 2 above)
Comment (amcjen) [inline Collections/Places/dataStore.js]: Fixed!  thanks!

**Pull request merged**